app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+# Top-level documentation comment for User module
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  def recent_posts(num)
+    posts.order(created_at: :desc).limit(num)
+  end
```

Comment:
- [ ] @Uthmanbello great job adding the recent_posts method, however, the method should only return the 3 most recent posts by default, in your case, you have made the number of posts a variable kindly make it a fixed value of 3.
![image](https://user-images.githubusercontent.com/61408860/236510551-b901f074-dcc7-44a9-9fbe-45845e7dee10.png)